src/main/java/org/tb/dailyreport/rest/WorkingDayRestEndpoint.java

```diff
@@ -0,0 +1,117 @@
+package org.tb.dailyreport.rest;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.server.ResponseStatusException;
+import org.tb.auth.AuthorizedUser;
+import org.tb.common.exception.AuthorizationException;
+import org.tb.common.exception.InvalidDataException;
+import org.tb.common.util.DateUtils;
+import org.tb.dailyreport.domain.Workingday;
+import org.tb.dailyreport.persistence.WorkingdayDAO;
+import org.tb.employee.persistence.EmployeecontractDAO;
+
+import java.time.LocalDate;
+
+import static java.util.Optional.ofNullable;
+import static org.springframework.http.HttpStatus.*;
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping(path = { "/api/workingday" })
```

Comment:
warum heißt die Klasse WorkingDayRestEndpoint und es ist auch in dem package "rest", aber der Pfad ist /api/workingday und nicht "/rest/workingday"?

Reply:
/api haben wir, da microsoft das bevorzugt in bestimmten setups. /rest sollten wir weiterhin auch haben, also beides

Reply:
/rest sollte hinzugefügt werden.

Reply:
vergleiche mit DailyReportRestEndpoint